examples/platform/silabs/SiWx917/uart.cpp

```diff
@@ -118,6 +118,30 @@ int16_t uartConsoleWrite(const char * Buf, uint16_t BufLength)
     return BufLength;
 }
 
+/**
+ * @brief Write Logs to the Uart. Appends a return character
+ *
+ * @param log pointer to the logs
+ * @param length number of bytes to write
+ * @return int16_t Amount of bytes written or ERROR (-1)
+ */
+int16_t uartLogWrite(const char * log, uint16_t length)
+{
+    if (log == NULL || length < 1)
+    {
+        return UART_CONSOLE_ERR;
+    }
+    for (int i = 0; i < length; i++)
+    {
+        Board_UARTPutChar(*log++);
```

Comment:
The `*log++` isn't quite optimal since you're adding an additionnal addition while being in a `for` loop.
```suggestion
        Board_UARTPutChar(log[i]);
```

Reply:
Made this change.